deploy/stack2/docker-compose.yml

```diff
@@ -11,11 +11,13 @@ services:
 
   api:
     restart: always
+    volumes:
+      - "/tmp:/tmp"
     build:
       context: ../../
       args:
         INCLUDE_UWSGI: 'true'
     environment:
       EXAMPLE_API_REVERSE_PROXY_SETUP: 'true'
       FLASK_CONFIG: 'production'
-    command: 'uwsgi --need-app --manage-script-name --mount /=app:create_app() --uwsgi-socket 0.0.0.0:5000'
+    command: 'uwsgi --need-app --manage-script-name --mount /=app:create_app() --uwsgi-socket 0.0.0.0:5000 --enable-threads --logger file:logfile=/tmp/flask-stack2.log,maxsize=104857600'
```

Comment:
1. `--enable-threads` should not be used unless there is a need as it hurts performance (it enables GIL).
2. Given that file logs are not that common in Docker environments I would rather prefer having the information about the logger in the README file. Could you document it there and revert the direct changes to the Dockerfile and docker-compose.yml?